src/utils/pack.js

```diff
@@ -163,8 +163,12 @@ export const getDataToSignForBalanceProof = (
   return dataToSign;
 };
 
-export const getDataToSignForNonClosingBalanceProof = message => {
-  const bpData = getDataToSignForBalanceProof(message, "UPDATE_BALANCE_PROOF");
+export const getDataToSignForNonClosingBalanceProof = (
+  message,
+  isLT = false
+) => {
+  const msgType = isLT ? MessageType.BALANCE_PROOF : "UPDATE_BALANCE_PROOF";
```

Comment:
@debrayperalta @mauricioirace we need to change this. We must use always "update_balance_proof" right?

Reply:
No, as it will break the other cases, because the signature of a BALANCE_PROOF and a UPDATE_BALANCE_PROOF are different by 1 number.
That 1 number is what makes or breaks that signature.
Changing it will break the signature of the balance proof

Reply:
i still thinking that we dont need that parameter and we can just use the padding of UPDATE_BALANCE_PROOF